src/ducks/HistoricalBalance/balanceView/BalanceView.js

```diff
@@ -48,7 +49,7 @@ const BalanceView = ({ address = '', assets = [], onChangeQuery }) => {
   const handleAssetsChange = assets => {
     const newState = {
       ...walletAndAssets,
-      assets: assets.map(asset => asset.value)
+      assets: assets
```

Comment:
try to decrease extra code: `assets: assets` -> `assets`. The same in `LOC 25, 26, 35`